bot.js

```diff
@@ -239,16 +239,34 @@ spawnPokemon = async (message, designatedChannel) => {
                     .setThumbnail(collected.first().author.avatarURL())
                     .setImage(generatedPKMNData.sprite);
                 let out = "```" + generateSpaces("LV." + generatedPKMNData.level, 6) + "| IV:" + generateSpaces(generateIVSummary(generatedPKMNData.ivs), 5) + "| " + generatedPKMNData.nature[0] + "```";
-                embed.addField("Info:", out);
+                embed.addField("Info:" + '\n' + "PokeDollars Obtained: " + parseInt(generatedPKMNData.level * 2),out, out);
+              //  embed.addField("PokeDollars Obtained:" + parseInt(generatedPKMNData.level * 2),out)
                 channelRef.send({embed});
 
+                /**
+                 * Handle Coins
+                 */
+
+                const snapshot = await db.collection('users').doc(collected.first().author.id).get().then((doc) => {
```

Comment:
Redundant query, there is already a call to get the user's data at the start of the collector's end event on line 185.